src/renderer/screens/config/ConfigScreen.scss

```diff
@@ -172,17 +173,12 @@
       align-items: flex-start;
       flex-direction: row;
       margin-bottom: 2rem;
+      justify-content: space-between;
     }
 
     .RowItem {
-      width: 50%;
+      width: calc(50% - 1rem);
```

Comment:
The expanding stack trace section of the error message was causing a jump. 50% was a lie due to css's padding rules. Using `calc` and `justify-content: space-between;` keeps things from jumping around (and makes the two columns actually equal in width)